dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/InputRendererType.java

```diff
@@ -50,5 +50,6 @@ public enum InputRendererType {
      * checkboxes with auto selected all
      * values defined in renderer options
      */
-    HIDDEN_VOCABULARY
+    HIDDEN_VOCABULARY,
+    PERIODO;
```

Comment:
My main takeaway from proposed solution is that it is really specific. In the wild there are many such kind of dictionaries from different disciplines and we are not getting closer to be able to support them. I am not sure if this is the direction that we want to go. Ideally I would like to see a solution that could work many different dictionaries that could be set up with dataverse and/or dataset fields configuration.

Reply:
Yest there will be more distionaries specific to domain, bou I think that trying to come with a generic solution up front is a faile attempt as we don't know about their potential structure and use cases. I think that once we have few of them implemented we can think about refactoring code into a more generic solution then. 

Anyway.. this distionary is activated only if the instance databese contains appropriate datasetfieldtype record so it is already anabled/disabled by configuration.

Reply:
Ok. If you think so.